chipy_org/apps/job_board/templates/job_board/job_post_form.html

```diff
@@ -0,0 +1,113 @@
+{% extends "job_board/base.html" %}
+    
+
+{% block body %}
+
+
+  {% if view_action == "create" %}
+      <h2>Create a Job Post</h2>
+  {% elif view_action == "update" %}
+      <h2>Update a Job Post</h2>
+  {% endif %}
+  
+
+<form action="." method="post">
+    {% csrf_token %}
+    
+    <table style="margin-bottom: 25px;"> 
+        
+        {% for field in job_user_form %}
+            <tr>
+                <td>{{field.label_tag}}</td>
+                <td>{{ field.errors }}{{field}}</td>
+            </tr>
+        {% endfor %}
+     
+        {% for field in job_post_form %}
+            {% if field == job_post_form.agree_to_terms %}
```

Comment:
It'd be great to figure out how to avoid this if breakout, but for now this makes a lot of sense with the workarounds needed on several fields.

Reply:
Agreed. :+1: